src/Altinn.AccessManagement.Core/Helpers/DelegationHelper.cs

```diff
@@ -488,6 +490,30 @@ public static List<RequestToDelete> GetRequestToDeleteResourceRegistryService(in
             };
         }
 
+        /// <summary>
+        /// Builds a RequestToDelete request model for revoking all delegated rules for a resource registry service
+        /// </summary>
+        public static List<RequestToDelete> GetRequestToDeleteResourceRegistryService(int authenticatedUserId, IEnumerable<AttributeMatch> resource, int fromPartyId, IEnumerable<AttributeMatch> to)
```

Comment:
Name is misleading as it's not only `ResourceRegistryService`.

Reply:
Renamed "GetRequestToDeleteResourceRegistryService" to "GetRequestToDeleteResource"